### DISCUSSION TITLE
Status M for files

### DISCUSSION THREAD

**Discussion by mafsi** [twpayne/chezmoi#2505] [Q&A] [answered]
I have some files that I added to chezmoi & they are tracked, and also they are in the repo. However, issuing `chezmoi status` I still get them with `M` flag not `MM`
```
$ chezmoi status
 M .config/polybar/scripts/power
 M .config/rofi/scripts/net-man-rofi
 M .config/rofi/scripts/rofi-edit-config.sh
```
What is the `M` flag & how can I get rid of them in the status?

**Comment by twpayne** [comment, marked as answer]
The `M` in the second column shows that there are differences between the file in your home directory and what chezmoi thinks the files should be.

Running `chezmoi diff` will show you the differences, which might include changes to their permissions.

Running `chezmoi apply` will make any changes and clear the statuses.

**Reply by mafsi** [reply]
Thanks a lot!
